<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>923</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>1221</width>
      <height>791</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="Inicio">
     <widget class="QPushButton" name="btnCERRAR">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>370</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Cerrar Programa</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonExercise1">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonExercise2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonExercise3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>210</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonExercise4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>290</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 4</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Ejercicio1">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnSumar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sumar Matrices</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnRestar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Restar Matrices</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnMultiplicar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Multiplicar Matrices</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sB_Col1">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QTableWidget" name="tW_tabla1">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>471</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHacerMatrices">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>60</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Hacer Matrices</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Columnas:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Filas:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sB_Fil1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QSpinBox" name="sB_Fil2">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Columnas:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sB_Col2">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Filas:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tW_tabla2">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>130</y>
        <width>451</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>130</y>
        <width>31</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>420</y>
        <width>971</width>
        <height>31</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTableWidget" name="tW_Resultado">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>440</y>
        <width>451</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <zorder>label_8</zorder>
     <zorder>BtnSumar</zorder>
     <zorder>BtnRestar</zorder>
     <zorder>BtnMultiplicar</zorder>
     <zorder>sB_Col1</zorder>
     <zorder>tW_tabla1</zorder>
     <zorder>label_21</zorder>
     <zorder>label_22</zorder>
     <zorder>sB_Fil1</zorder>
     <zorder>sB_Fil2</zorder>
     <zorder>label_23</zorder>
     <zorder>sB_Col2</zorder>
     <zorder>label_24</zorder>
     <zorder>tW_tabla2</zorder>
     <zorder>line</zorder>
     <zorder>line_2</zorder>
     <zorder>tW_Resultado</zorder>
     <zorder>btnHacerMatrices</zorder>
    </widget>
    <widget class="QWidget" name="Ejercicio2">
     <widget class="QSpinBox" name="num_buscar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QSpinBox" name="sB_Fil">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="Buscar_n">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Hacer Matriz</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tW_tabla">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>50</y>
        <width>481</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Columnas:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sB_Col">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="mostrar_info2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>430</y>
        <width>591</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese el numero del cual quierer saber su posicion: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Filas:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese el numero de Filas y Columnas que desea para la Matriz</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Ejercicio3">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Columnas:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese el numero de Filas y Columnas que desea para la Matriz</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tW_tabla4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>70</y>
        <width>481</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sB_Fil3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHacerMatrizTrans">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Hacer Matriz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Filas:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sB_Col3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTransponer">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>310</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Transponer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Ejercicio4">
     <widget class="QPushButton" name="TransponerMatriz">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rotar matriz</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sB_Fil_Col">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QTableWidget" name="tW_Mostrar">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>80</y>
        <width>581</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>FIlas/Columnas: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese un numero para Rotar la Matriz</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="BTNRETURN">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>0</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Regresar al Menú</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1257</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
